app/helpers/comments_helper.rb

```diff
@@ -49,6 +49,8 @@ def get_commenter_pseud_or_name(comment)
     if comment.pseud_id
       if comment.pseud.nil?
         ts("Account Deleted")
+      elsif comment.pseud.user.official
```

Comment:
We'd like the span containing the role to be outside the link. Sorry for not being totally clear about that in the issue.